src/dist/client_auth.rs

```diff
@@ -786,7 +780,8 @@ pub fn get_token_oauth2_code_grant_pkce(
     runtime
         .block_on(server.with_graceful_shutdown(async move {
             let _ = shutdown_signal.await;
-        } ))
+        } )).unwrap()
+        // TODO(drahnr): remove below when not needed anymore
```

Comment:
Avoid adding TODOs, add it as discussion in the PR :)

Reply:
The below should be uncommented, it's a stray development simplification.

Reply:
Small improvement(?) still wondering how to make it better so it compiles.

changes: 5ee8ed5